### PR TITLE
WF-768: document command change

### PR DESCRIPTION
**Pull request by frozenL**
document the command change: `spatial alpha config convert`

more to be added

### CODE REVIEW

File: docs/reference/launch-configuration-with-the-Platform-SDK.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+The Platform SDK currently only works with the CPL launch configuration format, so you need to convert your launch configuration file back to CPL format before you can use it with the Platform SDK. We’ve added a command that does this for you.
+
+## How do I use it?
+To convert an FPL launch configuration file to its CPL equivalent, run the following command inside the directory containing your FPL-format `spatialos.json`:
+```bash
+spatial alpha config convert
+```
+By default, this prints the result to `cpl_launch.json`. If you want to give the resulting file a different name or save it in a different place, you can set the `--output_path` flag to specify a filepath to the CPL launch configuration file:
+```bash
+spatial alpha config convert --output_path "<output_path>"
```
Comment (JonasImprobable) [inline docs/reference/launch-configuration-with-the-Platform-SDK.md]: Nit: Just to be consistent with our existing code snippets:
```suggestion
spatial alpha config convert --output_path <output-path>
```

File: docs/reference/launch-configuration-with-the-Platform-SDK.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+```bash
+spatial alpha config convert
+```
+By default, this prints the result to `cpl_launch.json`. If you want to give the resulting file a different name or save it in a different place, you can set the `--output_path` flag to specify a filepath to the CPL launch configuration file:
+```bash
+spatial alpha config convert --output_path <output-path>
+```
+You can now use this file with the Platform SDK.
+
+
```
Comment (JonasImprobable) [inline docs/reference/launch-configuration-with-the-Platform-SDK.md]: Unnecessary whiteline

Comment (frozenL) [reply docs/reference/launch-configuration-with-the-Platform-SDK.md]: thanks! done

**Comment by frozenL** [comment]
any idea where to put this file, and what's the filename?

File: docs/reference/launch-configuration-with-the-Platform-SDK.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Starting an FPL project using the Platform SDK
```
Comment (JonasImprobable) [inline docs/reference/launch-configuration-with-the-Platform-SDK.md]:
```suggestion
# Starting a deployment using the Platform SDK
```

File: docs/reference/launch-configuration-with-the-Platform-SDK.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+```bash
+spatial alpha config convert
+```
+By default, this prints the result to `cpl_launch.json`. If you want to give the resulting file a different name or save it in a different place, you can set the `--output_path` flag to specify a filepath to the CPL launch configuration file:
+```bash
+spatial alpha config convert --output_path <output-path>
+```
+You can now use this file with the Platform SDK.
+
+**Notes:** The current workflow is temporary, and this command is only available in `spatial alpha`. We will have a better workflow in the future.
```
Comment (JonasImprobable) [inline docs/reference/launch-configuration-with-the-Platform-SDK.md]: Put this higher up, e.g. in line 4.

**Comment by anne-edwards** [comment]
I guess this would fit best under Migrating (https://github.com/spatialos/FlexibleProjectExample#migrating-from-the-current-project-layout-cpl-to-the-flexible-project-layout-fpl) in the list of docs, although it doesn't really fit there... it depends if there are going to be any other similar docs relating to the Platform SDK (if there are, we can add a new heading, but if not then we might not want to add a heading for just this page).

File: docs/reference/launch-configuration-with-the-Platform-SDK.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+## How do I use it?
+To convert an FPL launch configuration file to its CPL equivalent, run the following command inside the directory containing your FPL-format `spatialos.json`:
+```bash
+spatial alpha config convert
+```
+By default, this prints the result to `cpl_launch.json`. If you want to give the resulting file a different name or save it in a different place, you can set the `--output_path` flag to specify a filepath to the CPL launch configuration file:
+```bash
+spatial alpha config convert --output_path <output-path>
+```
+You can now use this file with the Platform SDK.
```
Comment (JonasImprobable) [inline docs/reference/launch-configuration-with-the-Platform-SDK.md]: Can we put a link here to any existing docs?

File: docs/reference/launch-configuration-with-the-Platform-SDK.md (hunk 4 above)
Comment (anne-edwards) [reply docs/reference/launch-configuration-with-the-Platform-SDK.md]: Sorry @frozenL, I know I suggested moving it to the bottom! I don't mind where it goes - it's not crucial to the user at any particular point.

File: docs/reference/launch-configuration-with-the-Platform-SDK.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+The Platform SDK currently only works with the CPL launch configuration format, so you need to convert your launch configuration file back to CPL format before you can use it with the Platform SDK. We’ve added a command that does this for you.
+
+## How do I use it?
+To convert an FPL launch configuration file to its CPL equivalent, run the following command inside the directory containing your FPL-format `spatialos.json`:
+```bash
+spatial alpha config convert
+```
+By default, this prints the result to `cpl_launch.json`. If you want to give the resulting file a different name or save it in a different place, you can set the `--output_path` flag to specify a filepath to the CPL launch configuration file:
```
Comment (JonasImprobable) [inline docs/reference/launch-configuration-with-the-Platform-SDK.md]: By default, we will create a file called `cpl_launch.json` containing the converted launch configuration.

File: docs/reference/launch-configuration-with-the-Platform-SDK.md (hunk 5 above)
Comment (anne-edwards) [reply docs/reference/launch-configuration-with-the-Platform-SDK.md]: Good plan:

> `You can now use this file with the [Platform SDK](https://docs.improbable.io/reference/latest/platform-sdk/introduction).`

File: docs/reference/launch-configuration-with-the-Platform-SDK.md (hunk 4 above)
Comment (frozenL) [reply docs/reference/launch-configuration-with-the-Platform-SDK.md]: no worries! done

File: docs/reference/launch-configuration-with-the-Platform-SDK.md (hunk 5 above)
Comment (frozenL) [reply docs/reference/launch-configuration-with-the-Platform-SDK.md]: done

File: docs/migration-guide/configs-conversion-guide.md [hunk 7]
```diff
@@ -33,6 +33,8 @@ Below is a summary of how to populate the fields of an FPL worker configuration
 [Reference documentation for the SPL format](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config#launch-configuration-file)<br/>
 [Reference documentation for the FPL format](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/reference/launch-configuration.md)
 
+**Note:** If you want to use the [Platform SDK](https://docs.improbable.io/reference/latest/platform-sdk/introduction), you should **not** convert your launch configuration file into FPL format. The Platform SDK currently only works with SPL format launch configuration files.
```
Comment (JonasImprobable) [inline docs/migration-guide/configs-conversion-guide.md]: Can you add a note in the `Known issues` section in readme.me?

File: docs/reference/launch-configuration-with-the-Platform-SDK.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
+# Starting a deployment using the Platform SDK
+
+**Note:** The current workflow is temporary, and this command is only available in `spatial alpha`. We will have a better workflow in the future.
+
+The [flexible project layout (FPL) launch configuration file](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/reference/launch-configuration.md) has a different format from the [current project layout (SPL) launch configuration file](https://docs.improbable.io/reference/13.6/shared/reference/file-formats/launch-config#launch-configuration-file). You can find more details about the difference between the two formats [here](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/migration-guide/configs-conversion-guide.md#converting-the-launch-configuration-file).
```
Comment (JonasImprobable) [inline docs/reference/launch-configuration-with-the-Platform-SDK.md]:
```suggestion
The [flexible project layout (FPL) launch configuration file](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/reference/launch-configuration.md) has a different format from the [current project layout (SPL) launch configuration file](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config#launch-configuration-file). You can find more details about the difference between the two formats [here](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/migration-guide/configs-conversion-guide.md#converting-the-launch-configuration-file).
```

File: docs/reference/launch-configuration-with-the-Platform-SDK.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+The Platform SDK currently only works with the SPL launch configuration format, so you need to convert your launch configuration file back to SPL format before you can use it with the Platform SDK. We’ve added a command that does this for you.
+
+## How do I use it?
+To convert an FPL launch configuration file to its SPL equivalent, run the following command inside the directory containing your FPL-format `spatialos.json`:
+```bash
+spatial alpha config convert
+```
+By default, this prints the result to `spl_launch.json`. If you want to give the resulting file a different name or save it in a different place, you can set the `--output_path` flag to specify a filepath to the SPL launch configuration file:
```
Comment (JonasImprobable) [inline docs/reference/launch-configuration-with-the-Platform-SDK.md]: By default, we will create a file called `cpl_launch.json` containing the converted launch configuration.

Comment (frozenL) [reply docs/reference/launch-configuration-with-the-Platform-SDK.md]: done

File: docs/reference/launch-configuration-with-the-Platform-SDK.md (hunk 6 above)
Comment (frozenL) [reply docs/reference/launch-configuration-with-the-Platform-SDK.md]: done

**Comment by frozenL** [comment]
sounds good to merge? @anne-edwards @JonasImprobable

**Comment by frozenL** [comment]
Thanks both!

**Pull request merged**